submission/notifications/src/main/resources/templates/successful-resubmission.txt

```diff
@@ -1,9 +1,9 @@
 Dear ${USERNAME},
 
-Your BioStudies submission ${TITLE} with accession number ${ACC_NO} has been updated. You will be able to see the updated version at ${UI_URL}/studies/${ACC_NO} in the next 24 hours. ${RELEASE_MESSAGE}
+Your BioStudies ${TITLE} with accession number ${ACC_NO} has been updated. You will be able to see the updated version at ${UI_URL}/studies/${ACC_NO} in the next 24 hours. ${RELEASE_MESSAGE}
```

Comment:
Why are we removing the word "submission"? Maybe we should remove "BioStudies" instead, I think it'd look better:

"Your submission My Super Study with accession..."

Rather than

"Your Biostudies My Super Study with accession..."

Reply:
I am not changing  email content as part of ticket  I am not removing submission word but including as part for keywork {TITLE} as when there is not title email need to be "Your BioStudies submission with" rather than "Your BioStudies submission  with" (extra space product of replace title with an space. )